suma-project/Suma
# PHP 8.1 support

WIP: this is untested

Attempt to resolve issues that are causing Suma to fail with PHP 8.x. See the mailing list for examples.

* Update dependencies, particularly zf1-future and Gump.
* Remove `get_magic_quotes_gpc()` calls
* Fix phpcs complaints in our code
* Rebuild project
* Document how to use phpcs/PHPCompatibility
